client-cli/src/command.rs

```diff
@@ -385,7 +385,9 @@ impl Command {
                 block_height_ensure,
             } => {
                 let tendermint_client = WebsocketRpcClient::new(&tendermint_url())?;
-                let tx_obfuscation = get_tx_query(tendermint_client.clone())?;
+                let tendermint_client_tx_query = WebsocketRpcClient::new(&tendermint_url())?;
```

Comment:
Why two connections?

Reply:
if it cloned,  some disconnect message, occurs, 
i'll show it shortly

Reply:
remove two connections